setup.py

```diff
@@ -24,5 +24,5 @@
     ],
     packages=setuptools.find_packages(),
     python_requires=">=3.6",
-    install_requires="cocotb>=1.6.0,<2.0",
+    install_requires="cocotb>=1.6.0,<2.1",
```

Comment:
2.x will be compatible, you could make this `<3` instead.

Reply:
Thanks!  Would the only change then be to change setup to `install_requires="cocotb>=1.6.0,<3"`?